server/core/src/test/java/dev/slimevr/unit/TrackerTestUtils.kt

```diff
@@ -58,4 +59,13 @@ object TrackerUtils {
 			FastMath.isApproxEqual(q1.x, q2.x, tolerance) &&
 			FastMath.isApproxEqual(q1.y, q2.y, tolerance) &&
 			FastMath.isApproxEqual(q1.z, q2.z, tolerance)
+
+	fun randFloat(random: Random, minInclusive: Float, maxExclusive: Float): Float = minInclusive + (random.nextFloat() * (maxExclusive - minInclusive))
+
+	fun randQuat(random: Random): Quaternion = Quaternion(
+		randFloat(random, -1f, 1f),
+		randFloat(random, -1f, 1f),
+		randFloat(random, -1f, 1f),
+		randFloat(random, -1f, 1f),
+	).unit()
```

Comment:
just a comment but QuaternionTest has a guassian distribution-based random if interested

Reply:
I did see that, I figured this would be ideal to keep a seed per test rather than global? I'm not sure what is the best option

Reply:
The best option is not to use randoms in tests, but to cover fixed edge cases.

Reply:
Not randomized anymore :3